src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/JitHelper.cs

```diff
@@ -134,10 +134,10 @@ public static void GetEntryPoint(TypeSystemContext context, ReadyToRunHelper id,
                     break;
 
                 case ReadyToRunHelper.MemCpy:
-                    mangledName = "memcpy"; // TODO: Null reference handling
+                    methodDesc = context.GetHelperEntryPoint("MemoryHelpers", "MemCopy");
```

Comment:
@MichalStrehovsky Is a simple implementation of these new helpers needed in TestCoreLib too or can we live without them there?

Reply:
We can add them when someone runs into this. I'm not sure we have all the helpers to begin with.